index.Rmd

```diff
@@ -4,10 +4,10 @@ author: Joe Prego
  
 ---
 
-# Introduction
+# Introduction 
```

Comment:
Of course this is a first draft...but there are some small typos/sentence fragments to remember to remove. ("These distances will be..." and "ask" is missing a "k" )